code/modules/vending/cola.dm

```diff
@@ -20,8 +20,8 @@
 		           /obj/item/reagent_containers/food/drinks/soda_cans/monkey_energy = 1,
 		           /obj/item/reagent_containers/food/drinks/soda_cans/grey_bull = 1)
 	refill_canister = /obj/item/vending_refill/cola
-	default_price = 10
-	extra_price = 30
+	default_price = 75
+	extra_price = 200
```

Comment:
This means Nuka Cola and the like will cost 200 credits, right?

Reply:
That is a refill box that stocks the machine with many colas.
I believe it is 20 but I did not check, that would be 10-15 credits to a can or so.